models/data_warehouse.model.lkml

```diff
@@ -1960,7 +1960,18 @@ explore: nps_data {
   group_label: " Product: Messaging"
   description: "Contains raw NPS data. Typically you will want to use 'NPS User Monthly Score' instead of this, which contains a structured version of this data that allows you to trend the aggregate NPS score, not just the score at time of submission."
```

Comment:
@sri-byte any risk with making this visible for anyone to use?

Reply:
No, there is no risk involved. This data gets aggregated and used in the other view, which was built custom at the time.

Reply:
Thanks for confirming :+1: